notebooks/community/model_garden/model_garden_pytorch_stable_diffusion_xl_1_0.ipynb

```diff
@@ -0,0 +1,610 @@
+{
```

Comment:
<p>missing link to run in workbench</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2179/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Added the workbench link.</p>